src/udiskslinuxblock.c

```diff
@@ -1671,7 +1671,9 @@ has_whitespace (const gchar *s)
 static gchar *
 make_block_luksname (UDisksBlock *block, GError **error)
 {
+  udisks_linux_block_encrypted_lock (block);
```

Comment:
Umm, could you please move the `uuid` variable declaration above the code? This looks really weird :-)

Reply:
Fixed.